src/controllers/project.controller/updateProject/index.js

```diff
@@ -19,10 +21,27 @@ const updateProject = catchAsync(async (req, res) => {
     },
     changes: {},
   };
-  // if (req.body.caption) variables['changes']['caption'] = req.body.caption;
+
   if (req.body.description) variables.changes.description = req.body.description;
   if (req.body.title) variables.changes.title = req.body.title;
   if (req.body.link) variables.changes.link = req.body.link;
+  if (req.body.status !== undefined) variables.changes.status = req.body.status;
+  if (req.body.completed !== undefined) variables.changes.completed = req.body.completed;
+  if (req.body.looking_for_members !== undefined) variables.changes.looking_for_members = req.body.looking_for_members;
```

Comment:
Prameters from here on do not belong to the project table, you will have to update them on their separate tables like team, team_role_requirements etc